File: trail1/행 단위로 반복하여 출력/print-in-row.py
N = int(input())
board = [[0 for _ in range(N)] for _ in range(N)]
for i in range(N):
    for j in range(N):
        board[i][j] = j+1

for i in range(N):
    for j in range(N):
        print(board[i][j], end="")
    print()
        

File: trail1/행 단위로 반복하여 출력/print-in-row.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int N;
    cin >> N;
    vector<vector<int>> board (N, vector<int> (N, 0));
    for (int i=0; i<N; i++) {
        for (int j=0; j<N; j++) {
            board[i][j] = j+1;
        }
    }

    for (int i=0; i<N; i++) {
        for (int j=0; j<N; j++) {
            cout << board[i][j];
        }
        cout << "\n";
    }
    return 0;
}